spec/features/admin_deletes_all_comments_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.